src/patternfly/components/DataList/styles.scss

```diff
@@ -0,0 +1,199 @@
+
+  // List item border left
+  --pf-c-data-list__item--before--Color:                  var(--pf-global--active-color--100);
+  --pf-c-data-list__item--before--Width:                  var(--pf-global--BorderWidth--lg);
+  --pf-c-data-list__item--before--Transition:             var(--pf-global--Transition);
+  --pf-c-data-list__item--before--ZIndex:                 var(--pf-global--ZIndex--xl);
+
+  // Expandable content
+  --pf-c-data-list__expandable-content--PaddingTop:       var(--pf-global--spacer--xl);
+  --pf-c-data-list__expandable-content--PaddingBottom:    var(--pf-global--spacer--xl);
+  --pf-c-data-list__expandable-content--MaxHeight:        #{pf-size-prem(600px)};
+  --pf-c-data-list__expandable-content--Transition:       var(--pf-global--Transition);
+  --pf-c-data-list__expandable-content--BorderTopColor:   var(--pf-global--Color--light-200);
+  --pf-c-data-list__expandable-content--BorderTopWidth:   var(--pf-global--BorderWidth--sm);
+
+  // Expanded modifier
+  --pf-c-data-list--m-expanded--BoxShadow:                var(--pf-global--BoxShadow--md);
+  --pf-c-data-list--m-expanded--TransitionDuration:       var(--pf-global--TransitionDuration);
+
+  // spacers are a bit large on mobile view
+  // reduce spacer values for up to 380px
+  @media screen and (max-width: 380px) {
+    --pf-c-data-list__item--PaddingRight:                 var(--pf-global--spacer--lg);
+    --pf-c-data-list__item--PaddingLeft:                  var(--pf-global--spacer--lg);
+    --pf-c-data-list__toggle--MarginRight:                var(--pf-global--spacer--sm);
+    --pf-c-data-list__check--MarginRight:                 var(--pf-global--spacer--md);
+    --pf-c-data-list__actions--MarginLeft:                var(--pf-global--spacer--sm);
+  }
+
+  /* stylelint-enable */
+
+  list-style-type: disc;
+  background-color: var(--pf-c-data-list--BackgroundColor);
+}
+
+// li
+.pf-c-data-list__item {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  padding-right: var(--pf-c-data-list__item--PaddingRight);
+  padding-left: var(--pf-c-data-list__item--PaddingLeft);
+
+  // add border top to subsequent li's
+  & + & {
+    border-top: var(--pf-c-data-list__item--BorderWidth) solid var(--pf-c-data-list__item--BorderColor);
+  }
+
+  // left border treatment base
+  // color is transparent by default
+  &::before {
+    position: absolute;
+
+    // offset border positioning by parent border width
+    top: calc(var(--pf-c-data-list__item--BorderWidth) * -1);
+    left: 0;
+    z-index: var(--pf-c-data-list__item--before--ZIndex);
+    width: var(--pf-c-data-list__item--before--Width);
+    height: calc(100% + var(--pf-c-data-list__item--BorderWidth));
+    content: "";
+    background-color: transparent;
+    transition: var(--pf-c-data-list__item--before--Transition);
+  }
+
+  // expanded border treatment color
+  &.pf-m-expanded::before {
+    background-color: var(--pf-c-data-list__item--before--Color);
```

Comment:
looks like this should be `BackgroundColor`. not sure if it would be worth it for customization to create a var for the default `background-color` (L89 - `--pf-c-data-list__item--before--BackgroundColor`) and change this var to `--pf-c-data-list__item--m-expanded--before--BackgroundColor`?